gn3/computations/correlations.py

```diff
@@ -422,3 +423,60 @@ def fast_compute_tissue_correlation(primary_tissue_dict: dict,
     return sorted(
         tissues_results,
         key=lambda trait_name: -abs(list(trait_name.values())[0]["tissue_corr"]))
+
+
+def compute_correlation_2(corr_inputs):
+
+    (this_trait_samples, target_trait) = corr_inputs
```

Comment:
*issue*: pass in `samples` and `target_traits` as params; instead of a single set.  The resulting function should look like:
```suggestion
def compute_correlation_2(samples, target_trait):
```